Lib/test/test_gzip.py

```diff
@@ -687,6 +687,19 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
+def add_compress_level_flag(*compress_levels):
```

Comment:
11 lines just to pass a flag? IMHO it's too much, it can be done with less code: see below.